tests/cli_test.py

```diff
@@ -56,6 +57,203 @@ def cli11_optional_option_arg_test():
                                       '-c', chain, '-k', key, '{}'])
     assert(b'signatures' in output)
 
+    """Test option --abi-file """
```

Comment:
Place those new tests in a separate function/test-case.

Reply:
Fixed.